### PR TITLE
Mirror of BT in DSA450 - Python solution

### LINKED ISSUE #246: Python solution  for Mirror of Binary Tree in DSA 450
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. 

**Describe the solution you'd like**

**Additional context**
Link to the problem : https://practice.geeksforgeeks.org/problems/mirror-tree/1/

### PR DESCRIPTION
**Pull request by dsrao711**
Closes #246

### CODE REVIEW

File: Dynamic Programming/MirrorOfBinTree.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        
+        
+
+#{ 
+#  Driver Code Starts
+#Initial Template for Python 3
+
+
+
+#Contributed by Sudarshan Sharma
```
Comment (smv1999) [inline Dynamic Programming/MirrorOfBinTree.py:46]: Why is it given in some other's name ? Please look into this.

**Review by smv1999 — CHANGES_REQUESTED** [review]

File: Dynamic Programming/MirrorOfBinTree.py (hunk 1 above)
Comment (dsrao711) [reply Dynamic Programming/MirrorOfBinTree.py:46]: Hey so this question is solved on GFG , So we are just supposed to write the function part . The driver code is contributed by someone else. Ill edit and re submit

Comment (smv1999) [reply Dynamic Programming/MirrorOfBinTree.py:46]: Okay !

**Comment by dsrao711** [comment]
Done

**Review by smv1999 — APPROVED** [review]

**Pull request merged**